versioned_sidebars/version-v2.18.x-sidebars copy.json

```diff
@@ -0,0 +1,1099 @@
+          label: "Preparing for installation",
+          link: {type: "doc", id: "user-guide/installandconfig"},
+          items: [
+            "user-guide/zos-components-installation-checklist",
+            "user-guide/systemrequirements-zos",
+            "user-guide/install-nodejs-zos",
+            "user-guide/address-security-requirements",
+            "user-guide/address-authentication-requirements",
+            "user-guide/configure-uss",
+            "user-guide/address-storage-requirements",
+            "user-guide/address-network-requirements",
+            "user-guide/address-browser-requirements",
+          ]
+        },
+        {
+        type: "category",
+        label: "Installing Zowe Server Install Wizard",
+        link: {type:"doc", id: "user-guide/install-zowe-server-install-wizard"},
+        items: [
+          "user-guide/troubleshooting-zowe-server-install-wizard",
+        ]
+        },  
+        {
+          type: "category",
+          label: "Installing Zowe via SMP/E",
+          link: {type:"doc", id: "user-guide/install-zowe-smpe-overview"},
+          items: [
+            "user-guide/install-zowe-smpe",
+          ]
+        },
+        {
+          type: "category",
+          label: "Installing Zowe via z/OSMF from PSWI and SMP/E workflow",
+          link: {type:"doc", id:"user-guide/zosmf-install"},
+          items: [
+            "user-guide/install-zowe-pswi-address-requirements",
+            "user-guide/systemrequirements-zosmf",
+            "user-guide/systemrequirements-zosmf-lite",
+            {
+              type: "category",
+              label: "Installing Zowe via z/OSMF from PSWI",
+              link: {type:"doc", id:"user-guide/install-zowe-pswi"},
+              items: [
+                "user-guide/install-zowe-pswi-acquire",
+                "user-guide/install-zowe-pswi-deployment",
+              ],
+              },
+              "user-guide/install-zowe-smpe-zosmf-workflow",
+            ],
+        },
+        "user-guide/install-zowe-zos-convenience-build",
+        {
+          type: "category",
+          label: "Installing Zowe via a containerization build (PAX file)",
+          link: {type:"doc", id:"user-guide/k8s-introduction"},
+          items: [
+            "user-guide/k8s-prereqs",
+            "user-guide/k8s-downloading",
+            "user-guide/k8s-config",
+            "user-guide/k8s-using",
+          ],
+        },
+		    {
+          type: "category",
+          label: "Configuring",
+          link: {type:"doc", id:"user-guide/configuring-overview"},
+          items: [
+            {
+              type: "category",
+              label: "Initializing Zowe z/OS runtime",
+              link: {type:"doc", id:"user-guide/configure-zowe-runtime"},
+              items: [
+                "user-guide/initialize-zos-system",
+                "user-guide/zwe-init-subcommand-overview",
+                "user-guide/configuring-zowe-via-jcl",
```

Comment:
And of course, it will need to be indexed in the `version-v2.18.x-sidebars.json`

The combination of the changes laid out in the comments above shoudl resolve the issue as reported.
- #4265

One more thing, The issue mentions that it's possible that some content may be applicable to v2.17. However, I do not believe it's necessary for us to update older version of V2 (i.e. 2.17, 2.16, ...) given that 2.18.x is already in maintenance mode and developers won't be porting fixes to older versions of V2 either.

<img width="565" alt="image" src="https://github.com/user-attachments/assets/40e30573-f1ba-4b9c-84ab-245242a9579a" />